src/Microsoft.TemplateEngine.Cli/PostActionProcessors/AddProjectsToSolutionPostAction.cs

```diff
@@ -36,22 +36,23 @@ public bool Process(IEngineEnvironmentSettings environment, IPostAction actionCo
                 return false;
             }
 
-            Dotnet addProjToSlnCommand = Dotnet.AddProjectsToSolution(nearestSlnFilesFould[0], projectFiles);
+            string solutionFolder = GetSolutionFolder(actionConfig);
+            Dotnet addProjToSlnCommand = Dotnet.AddProjectsToSolution(nearestSlnFilesFould[0], projectFiles, GetSolutionFolder(actionConfig));
             addProjToSlnCommand.CaptureStdOut();
             addProjToSlnCommand.CaptureStdErr();
-            environment.Host.LogMessage(string.Format(LocalizableStrings.AddProjToSlnPostActionRunning, nearestSlnFilesFould[0], string.Join(" ", projectFiles)));
+            environment.Host.LogMessage(string.Format(LocalizableStrings.AddProjToSlnPostActionRunning, addProjToSlnCommand.Command));
```

Comment:
we should avoid string.Format, consider using string concatenation instead if possible

Reply:
applicable for any other new uses of `string.Format()`

Reply:
@donJoseLuis: it's not possible to use `string.concat` here - parameters in localized strings are in the middle of sentence and potentially can change the order in different locales.